docs/api_v4.md

```diff
@@ -46,6 +46,11 @@ If a `historic=1` param is included in the request, one additional field will be
 |-------------:|:---------------------------------------------|:------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `history`    | array of numbers                             | never                                                 | Ordered durations of all previous runs. The first item is the first run recorded by the runner's timer into the source file. The last item is the most recent one. This field is only nonempty if the source timer records history.           |
 
+If an `Accept` header is present, splits.io will try to render the run file natively not in JSON of the type specified.  The format for the header is `application/timer`, with timer being any of the timers that can be exported by splits.io.
```

Comment:
We should list all the valid Accept headers. If we add support for another timer it'll be more work to update this (and possibly confusing if we forget to), but the work saved for readers by having a usually-complete list here will far outweigh that.

Reply:
Do you have a preferred format for the list of acceptable values?

Reply:
I tried adding a table cause it looked snazzy to me, let me know if it's actually good looking or if I'm crazy.

Reply:
Yeah this looks good. There's a couple of sentences above it that restate what the table says that can probably be removed, overall this is a good format for this info though.